Assets/MRTK/Examples/StandardAssets/Models/Model_Bucky.fbx.meta

```diff
@@ -1,87 +0,0 @@
-fileFormatVersion: 2
-guid: 24d47aad909b7114f99ea8657d2883d8
```

Comment:
Does this need to be updated anywhere? Looks like it's being deleted from the Examples StandardAssets

Reply:
I wonder if it'd be better to leave this meta around and delete the ET-specific one instead. This one feels like it's in a more used location?

Reply:
let me look into that one

Reply:
So we DO have two copies of this model in the tree.... 

SDK\Features\UX\Prefabs\ProgressIndicators\ProgressIndicatorRotatingObject.prefab accesses it from the foundation pacakge.

I'll consolidate the refs

Reply:
fixed. updated the refs in the ET scene to use the consolidated files.